runtime/tchannel_outbound_call.go

```diff
@@ -89,15 +88,15 @@ func (c *tchannelOutboundCall) logFields(ctx context.Context) []zapcore.Field {
 		zap.Time("timestamp-finished", c.finishTime),
 	}
 
-	for k, v := range c.reqHeaders {
-		fields = append(fields, zap.String(
-			fmt.Sprintf("%s-%s", logFieldClientRequestHeaderPrefix, k), v,
-		))
-	}
-	for k, v := range c.resHeaders {
-		fields = append(fields, zap.String(
-			fmt.Sprintf("%s-%s", logFieldClientResponseHeaderPrefix, k), v,
-		))
+	var logFields []zap.Field
+	if c.client != nil && c.client.contextExtractor != nil {
+		headers := map[string]string{}
+		for k, v := range c.reqHeaders {
```

Comment:
This only includes request headers, what about response headers?
Also we need the prefix to differentiate the header keys between client and endpoint